assets/src/edit-story/components/canvas/editLayer.js

```diff
@@ -93,7 +98,7 @@ function EditLayerForElement({ element }) {
         }
       }}
     >
-      <EditPageArea ref={pageAreaRef} showOverflow={editModeGrayout}>
+      <EditPageArea ref={pageAreaRef} showOverflow={showOverFlow}>
```

Comment:
I think we should actually always show overflow. I can't think of any case where we wouldn't want it.